app/services/stories_api/v3/endpoints/story_items.rb

```diff
@@ -36,24 +38,14 @@ def post
           return @errors if @errors
           return create_error('MandatoryParamMissing', param: :item) unless params[:item]
 
-          position = params[:item].delete(:position)
+          position = @item_params.delete(:position)
 
           validator = StoriesApi::V3::Schemas::StoryItem::BlockValidator.new.call(params[:item])
           return create_error('SchemaValidationError', errors: validator.messages(full: true)) unless validator.success?
 
-          item_params = params[:item].deep_symbolize_keys
-
-          # FIXME: This is a temporary fix
-          # Can be removed after user_sets is retired
-          if item_params[:content][:id]
-            item_params[:record_id] = item_params[:content][:id]
-            record = SupplejackApi.config.record_class.custom_find(item_params[:record_id])
-            item_params[:content][:image_url] = record.large_thumbnail_url || record.thumbnail_url if record
-          end
-
-          story_item = story.set_items.build(item_params)
+          # FIXME: record_id is being posted in params by dnz website but not showing up here.
```

Comment:
Does `record_id` need to be added in here?

https://github.com/DigitalNZ/supplejack_api/blob/19dc09c2ba4b7f7a777c23a93ba9548b97f148ac/app/services/stories_api/v3/schemas/story.rb

Reply:
I don't believe so.  Look at line 49 of what I deleted.   It as working before, and works now.